javaslang/src-gen/test/java/javaslang/APITest.java

```diff
@@ -230,6 +230,26 @@ public void shouldLazyReturnNotNull() {
         assertThat(lazy(() -> 1)).isNotNull();
     }
 
+    @Test
+    public void shouldOptionReturnNotNull() {
+        assertThat(option(1)).isNotNull();
```

Comment:
could you provide a test for `option(null)` also, I'm not sure how that should behave differently from `some` (I guess it should represent a `none` intuitively)